lib/src/cli.rs

```diff
@@ -512,6 +515,27 @@ async fn usroverlay() -> Result<()> {
         .into());
 }
 
+/// Implementation of `bootc build commit`
+/// async fn lint() -> Result<()> {
+fn lint() -> Result<()> {
+    // TODO: Check if 1 or more folder under the /lib/modules/ directory
+    println!("Checking if we are in the container");
+    if ostree_ext::container_utils::is_ostree_container()? {
+        let modules_path = cap_std::fs::Dir::open_ambient_dir("lib/modules", cap_std::ambient_authority())?;
+        for entry in modules_path.entries()? {
+            let dir = entry?;
+            let fileName= dir.file_name();
+            debug(fileName);
```

Comment:
First nit: naming scheme in Rust is snake_case and not camelCase, i.e. should be `file_name` or `filename` or just `name`.

I think you want either `tracing::debug!` here (suitable to leave in persistently, will only show up with `RUST_LOG=debug`) or just a plain `dbg!` (to print something while testing/developing, but can't go in committed code).